packages/react-dom/src/events/__tests__/BeforeInputEventPlugin-test.js

```diff
@@ -810,9 +812,12 @@ describe('BeforeInputEventPlugin', () => {
     testContentEditableComponent(environments[1], scenarios);
   });
 
-  it('should extract onBeforeInput when simulating in env with no CompositionEvent on input[type=text]', () => {
-    testInputComponent(environments[2], scenarios);
-  });
+  // TODO investigate why this test fails for Fire
```

Comment:
This test seems to pass. ChangeEventPlugin-test.internal.js is red though

Reply:
It's failing for me. Note, the way to run React Fire tests on this PR is: `yarn test-fire` or `yarn test-fire-prod`.